jolie/src/main/java/jolie/monitoring/events/OperationCallEvent.java

```diff
@@ -34,7 +34,7 @@ public class OperationCallEvent extends MonitoringEvent {
 
 
 	public OperationCallEvent( String operationName, String processId, String messageId, int status, String details,
-		String outputPort, Value message ) {
+		String outputPort, Value message, String rawId ) {
```

Comment:
`rawId` is not descriptive. Could you suggest a better name? `messageId` should probably become `requestId` and `rawId` might become `messageId`.

Reply:
(Same comment for the Jolie types corresponding to this.)

Reply:
@fmontesi @klag 
I used the convention decided in our last conversation.
In the db  associated with Monitor, I also created the same identifier, and rawId is userd in isJolieLoggingMonitor.

Next time perhaps is better working with a written analysis.

Reply:
I do not recall saying that `rawId` would be fine (and I'd be surprised at me if I did). Sorry if I did. Any proposal for alternative? I think that `rawId` is going to be confusing to you as well going forward. `raw` is misleading. We have internal message ids (what you call rawId) and request ids.

Reply:
Hi do you think protocolMessageId would be nice?

Reply:
@fmontesi could we decide the name of the field as soon as possible? It has impacts on some code on our parts

Reply:
Mmh, `protocolMessageId` makes it seems related to the protocol, but every message has it regardless of the protocol.
What about making `messageId` the actual id of the message (instead of the request id as of now), and then a new `requestId` field that contains the request id? Would this be changing too much in existing implementations of monitors?

Reply:
Since messageId alerady exixts we should change two variables instead of one. 
I suggest to avoid to alter the current messageId leaving this modification for major revision in the future.

rawMessageId?

Reply:
Sounds good, let's avoid changing too much in one step if it causes problems.
I suggest `internalMessageId` then. `raw` seems to be "low-level" but it's actually on the same level, just different.